code/datums/outfits/ert/mercenary.dm

```diff
@@ -33,7 +33,7 @@
 /datum/outfit/admin/ert/mercenary/specialist
 	name = "Mercenary Freelancer Medic"
 
-	belt = /obj/item/storage/belt/medical
+	belt = /obj/item/storage/belt/medical/first_responder/combat
 	gloves = /obj/item/clothing/gloves/latex
 
 	backpack_contents = list(
```

Comment:
Could probably move the theta bottle and coagzolug injectors to the belt now for these loadouts, but that's basically just extra QoL.

Reply:
True